### ISSUE TITLE
Inverted Y Axis

### ISSUE THREAD

**Issue by antunesl** [Mapsui/Mapsui#423]
In a Xamarin Forms project using Mapsui (v1.4.2) the map is rendered with the inverted Y axis (images in the end of the issue), **only in iOS**. In the moment I can't explain what is causing this behavior, because I have [other project](https://github.com/antunesl/mapsui-forms-sample) (a sample project that I use for testing) where this doesn't happen.
I simplified the map initialization to be able to compare to the sample project where it works, in the moment I'm only adding the OpenStreetMap tile layer. I suspect that this has to be related to the tile fetching/rendering, but I'm not sure why it works on the other project.

**Mapsui Initialization**
```c#
_mapView = new MapsUIView();
_mapView.NativeMap.Layers.Add(OpenStreetMap.CreateTileLayer());
```

**The MapsUIView view**
(used as a wrapper for the **Mapsui.UI.iOS.MapControl** used in the custom renderer)
```c#
public class MapsUIView : Xamarin.Forms.View
{
    public Mapsui.Map NativeMap { get; set; }

    protected internal MapsUIView()
    {
        NativeMap = new Mapsui.Map
        {
            BackColor = Color.White
        };
    }
}
```

**iOS Custom Map Render**
(Is exactly the same in the two projects)
```c#
[assembly: ExportRenderer(typeof(MapsUIView), typeof(MapViewRenderer))]
namespace Application.iOS
{
    [Preserve(AllMembers = true)]
    public class MapViewRenderer : ViewRenderer<MapsUIView, Mapsui.UI.iOS.MapControl>
    {
        Mapsui.UI.iOS.MapControl mapNativeControl;
        MapsUIView mapViewControl;

        protected override void OnElementChanged(ElementChangedEventArgs<MapsUIView> e)
        {
            base.OnElementChanged(e);

            if (mapViewControl == null && e.NewElement != null)
                mapViewControl = e.NewElement;

            if (mapNativeControl == null && mapViewControl != null)
            {
                var rectangle = mapViewControl.Bounds;
                var rect = new CGRect(rectangle.X, rectangle.Y, rectangle.Width, rectangle.Height);

                mapNativeControl = new Mapsui.UI.iOS.MapControl(rect)
                {
                    Map = mapViewControl.NativeMap,
                    Frame = rect
                };

                SetNativeControl(mapNativeControl);
            }
        }
    }
}
```

**Inverted Map**
<img width="345" alt="map_1" src="https://user-images.githubusercontent.com/1543421/40613877-40cacf7e-6278-11e8-9bb4-e5f61cd8cb4b.PNG">

**Normal Map**
<img width="345" alt="map_2" src="https://user-images.githubusercontent.com/1543421/40613883-448d896c-6278-11e8-95c1-84c3e4360724.PNG">

**Comment by antunesl** [comment]
I updated the Mapsui reference to **v2.0.0-beta.2**, and that seems to solve the issue in this case. Currently is not a solution to me because I need the #421 fix present in 1.4.2 and not in 2.0.0-beta.2.
Another thing I can't explain it's the sample project that is working ok with the 1.4.2 version.

**Comment by pauldendulk** [comment]
I don't think this problem is related to Mapsui. It seems the entire skia canvas is inverted since de osm attribution text is inverted too. We don't do something like this in Mapsui. 

It won't be too long before we publish another 2.0 beta with the 421 fix, perhaps you should wait for this.

I am interested to learn what causes your problem though.

**Comment by antunesl** [comment]
Thanks, I'm gonna wait for that 2.0 beta (i can't wait for too long in this project unfortunately), even if I can't wait and need to go with other solution I'll still test this when the version comes out.

**Comment by pauldendulk** [comment]
What happens if you create a new xamarin forms project with just the mapsui map?

**Comment by antunesl** [comment]
> What happens if you create a new xamarin forms project with just the mapsui map?

I began suspecting the project dependencies. I created an empty Xamarin Forms project like you suggested, and start adding all the dependencies I have in the other project, one by one.

I was right, the dependency from '**Xamarin.FFImageLoading.Svg.Forms**' was inverting the map. I think it's because it depends on **SkiaSharp**, and somehow a bad SkiaSharp version was being chosen at build time.

The build log had this line:
```
Consider app.config remapping of assembly "SkiaSharp, Culture=neutral, PublicKeyToken=0738eb9f132ed756" 
        from Version "1.59.0.0" [(...)\repos\MapsuiTest_FormsVanilla\MapsuiTest_FormsVanilla.iOS\bin\iPhoneSimulator\Debug\SkiaSharp.dll] 
          to Version "1.60.0.0" [(...)\.nuget\packages\skiasharp\1.60.0\lib\XamariniOS\SkiaSharp.dll] to solve conflict and get rid of warning.
```

Right now, I have the 1.4.2 version working ok. I think this is not a bug, but maybe a possible incompatibility with a specific version of SkiaSharp.

Thanks for the help Paul!
